programs/scholarship.md

```diff
@@ -28,23 +27,23 @@ The scholarship committee will review each application holistically, taking into
   <tbody>
     <tr>
       <td>Application release</td>
-      <td>February 27, 2016 (at Kickoff)</td>
+      <td>March 4, 2017 (at Kickoff)</td>
```

Comment:
@quantumfusion Did you want to update the date for Kickoff (3/11)?  Or do you want to leave it as 3/4